impl/src/test/java/org/ehcache/internal/store/offheap/OffHeapValueHolderPortabilityTest.java

```diff
@@ -56,16 +59,23 @@ public void testEncodeDecode() {
   }
 
   @Test
-  public void testWriteBackSupport() {
+  public void testWriteBackSupport() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
     ByteBuffer encoded = valueHolderPortability.encode(originalValue);
     WriteContext writeContext = mock(WriteContext.class);
     OffHeapValueHolder<String> decoded = valueHolderPortability.decode(encoded, writeContext);
 
+    Class<?> abstractValueHolder = AbstractValueHolder.class;
+    Method setHits = abstractValueHolder.getDeclaredMethod("setHits", long.class);
+    setHits.setAccessible(true);
+    
     decoded.setExpirationTime(4L, TimeUnit.MILLISECONDS);
     decoded.setLastAccessTime(6L, TimeUnit.MILLISECONDS);
+    setHits.invoke(decoded, 8L);
```

Comment:
The test had to be slightly changes as `AbstractValueHolder.setHits()` was made protected. Also, with one of the set methods made protected should the others be made protected as well, namely, `setExpirationTime()` and `setLastAccessTime()`